lib/core/DocsLayout.js

```diff
@@ -116,7 +116,7 @@ class DocsLayout extends React.Component {
               )}
           </Container>
           {hasOnPageNav && (
-            <nav className="onPageNav">
+            <nav className="onPageNav docOnPageNav">
```

Comment:
Same question regarding this line.

Reply:
because the class is used in other page, like blog so it may break the other page if i modify it directly